src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -199,8 +199,8 @@
     <script type="text/template" id="template-legend-item-multiple">
         <div class="legend-layer multiple">
             <div class="legend-title"><%- layerName %></div>
-            <% _.each(legend, function(item) { %>
-                <div class="item">
+            <% _.each(legend, function(item, index) { %>
+                <div class="item <% if (index > 4) { %>extra<% } %>">
```

Comment:
I have this working. Here's a simple (but maybe ineffective) way to show there are missing breaks in the middle:

![screen shot 2016-01-19 at 9 16 45 am](https://cloud.githubusercontent.com/assets/1042475/12420929/6ef59406-be8d-11e5-89d8-2000ed0eef03.png)